samples/crypto/sha256/src/main.c

```diff
@@ -33,12 +35,15 @@ LOG_MODULE_REGISTER(sha256, LOG_LEVEL_DBG);
 /* ====================================================================== */
 /*				Global variables/defines for the SHA256 example			  */
 
-#define NRF_CRYPTO_EXAMPLE_SHA256_TEXT_SIZE (100)
+#define NRF_CRYPTO_EXAMPLE_SHA256_TEXT_SIZE (150)
 #define NRF_CRYPTO_EXAMPLE_SHA256_SIZE (32)
+#define NRF_CRYPTO_EXAMPLE_SHA256_CHUNK_SIZE (25)
 
 /* Below text is used as plaintext for computing/verifying the hash. */
 static uint8_t m_plain_text[NRF_CRYPTO_EXAMPLE_SHA256_TEXT_SIZE] = {
 	"Example string to demonstrate basic usage of SHA256."
+	"With a bit longer input stream, that shows that nRF "
+	"Chips can even hash long messages."
```

Comment:
```suggestion
	"Chips can even hash long messages."
```
that's a public sample,  please remove the "can even hash long messages". Make it a bit more boring please.